src/components/PeopleFilters.tsx

```diff
@@ -1,12 +1,55 @@
+import classNames from 'classnames';
+import { useSearchParams } from 'react-router-dom';
+import { SearchLink } from './SearchLink';
+import { FilterParam } from '../types/FilterParams';
+import { Sex } from '../types/Sex';
+import { getSearchWith } from '../utils/searchHelper';
+
 export const PeopleFilters = () => {
+  const CENTURIES = ['16', '17', '18', '19', '20'];
```

Comment:
It is better to define this constant outside the component so that it is not created every time a new render is performed.